docbook/reference/en/en-US/modules/Jaxb.xml

```diff
@@ -2,7 +2,10 @@
    <title>JAXB providers</title>
 
    <para>
-
+      NOTE: This section contains some outdated information about the deprecated Jettison provider, and the relative information will be
+      replaced with the usage of currently supported Jackson2 provider.
+   </para>
+   <para>
```

Comment:
Hi @liweinan,

Now that you have removed references to jettison, this section DOESN'T have outdated information any more. I would just say that the jettison provider is no longer supported.

Reply:
let me remove this note. And the deprecation of the jettison provider I guess should be announced in release note. In addition it's written in our blog post: https://developer.jboss.org/en/resteasy/blog/2019/02/12/jettison-has-been-removed-on-master-branch